pkg/validator/output.go

```diff
@@ -211,7 +212,7 @@ func (res ResultSet) GetPrettyOutput() string {
 			}
 		}
 		if color.NoColor {
-			status = status[2:] // remove emoji
+			status = strings.Fields(status)[1] // remove emoji
```

Comment:
I wonder if this would be safer as
```
strings.Join(strings.Fields(status)[1:], " ")
```

Reply:
Probably not a big deal

Reply:
@rbren Please, feel free to adjust it to whatever serves the project best :)